src/DefinedTermSet.src.json

```diff
@@ -0,0 +1,38 @@
+{
+    "type": "DefinedTermSet",
+    "uri": "http://schema4i.org/DefinedTermSet",
+    "description": "A set of defined terms, for example a set of categories or a classification scheme, a glossary, dictionary or enumeration.",
+    "links": [],
```

Comment:
Please add `https://schema.org/DefinedTermSet`as link to the original schema type.